client/a8c-for-agencies/sections/agency-tier/overview-content/influenced-revenue.tsx

```diff
@@ -0,0 +1,34 @@
+import { formatCurrency } from '@automattic/number-formatters';
+import { __ } from '@wordpress/i18n';
+import { Stat } from 'calypso/dashboard/components/stat';
+import { ALL_TIERS } from './constants';
+import type { AgencyTier } from './types';
+
+export default function InfluencedRevenue( {
+	currentAgencyTier,
+	totalInfluencedRevenue,
+}: {
+	currentAgencyTier?: AgencyTier;
+	totalInfluencedRevenue: number;
+} ) {
+	const currentTier = ALL_TIERS.find( ( tier ) => tier.id === currentAgencyTier );
```

Comment:
Perhaps we can use `currentAgencyTierId` as `currentAgencyTier` to avoid confusion about which is which. Then we can use `currentAgencyTier` as `currentTier`. wdyt?